### PR TITLE
Refactor TagsCheck.py

### PR DESCRIPTION
**Pull request by thisisshub**
This commit contains,
- Refactor checks to individual methods
    - Convert %s strings to {}.format methods
    - Rename variables to snake-case-naming style
- Modify Docstrings and methods in TagsCheck.py
- Drop checks no-packager, invalid-packager
     - Modify methods and Docstrings

### CODE REVIEW

File: rpmlint/checks/TagsCheck.py [hunk 1]
```diff
@@ -304,33 +537,50 @@ def check(self, pkg):
...
+                # Check if a package contians a entry in %changelog
+                # with timestamp thats in the future of its writing
                 elif clt > time.time():
                     self.output.add_info('E', pkg, 'changelog-time-in-future',
                                          time.strftime('%Y-%m-%d', time.gmtime(clt)))
 
+    def _check_license(self, pkg, rpm_license):
+        """Trigger check no-license, invalid-license-exception, invalid-license
+
+        Checks are triggered due to the configuration set by the user in the confifdefaults.toml
```
Comment (Conan-Kudo) [inline rpmlint/checks/TagsCheck.py:575]: `confifdefaults.toml` -> `configdefaults.toml`

**Comment by Conan-Kudo** [comment]
@thisisshub I am actually using the packager tag check for linting spec files building for Debian/Ubuntu, where the `Packager` tag is mandatory, so please don't drop it.

**Comment by thisisshub** [comment]
@Conan-Kudo @scarabeusiv 
Is there anything else you'd like me to change? otherwise, I think this PR can be merged ^^

**Comment by scarabeusiv** [comment]
The lgtm tells you there is stored variable thats not used anywhere.

That probably means that the function split is missing some feature.

**Comment by kstreitova** [comment]
> @thisisshub I am actually using the packager tag check for linting spec files building for Debian/Ubuntu, where the `Packager` tag is mandatory, so please don't drop it. Just disable running the check by default in `configdefaults.toml`.

I think that Debian/Ubuntu rules shouldn't be reasons to hold that code. The main purpose of rpmlint is to lint RPMS and we just can't take other non-rpm distros into account. That doesn't make any sense to me and it sounds like a very non-systematic way.

I think we should just drop it as all RPM based distros we take into consideration during the whole rpmlint rewrite (like Fedora, openSUSE, Mageia, ...) don't use Packager tag anymore.

**Comment by Conan-Kudo** [comment]
@thisisshub Can you squash your last two commits and rewrite it to just be about the typo fix?

**Comment by thisisshub** [comment]
> @thisisshub Can you squash your last two commits and rewrite the commit message to just be about the typo fix?

I shall but after the upstream decides whether to keep or drop those checks since @kstreitova makes a valid point. Since Debian is a pkg distro or a non-rpm distro. I too am in favor of dropping these checks since they are no longer needed.

**Pull request merged**